packages/@sanity/types/src/schema/definition/type/array.ts

```diff
@@ -11,13 +11,10 @@ import {type BaseSchemaDefinition, type SearchConfiguration, type TitledListValu
 /** @public */
 export interface ArrayOptions<V = unknown> extends SearchConfiguration {
   list?: TitledListValue<V>[] | V[]
-  /**
-   * layout: 'tags' only works for string array
-   * layout: 'grid' only works for arrays with objects
-   */
   // inferring the array.of value for ArrayDefinition cause too much code-noise and was removed.
   // Since we don't have the type-info needed here, we allow values
-  layout?: 'tags' | 'grid'
+  layout?: 'list' | 'tags' | 'grid'
+  // @deprecated This option does have any effect anymore
```

Comment:
One tiny thing: I think this needs to be a block comment to have an effect.

```suggestion
  /** @deprecated This option does have any effect anymore */
```

Reply:
Fixed!